velox/exec/RowContainer.cpp

```diff
@@ -427,18 +442,36 @@ void RowContainer::freeVariableWidthFields(folly::Range<char**> rows) {
 void RowContainer::checkConsistency() const {
```

Comment:
Maybe change this to a test method? This is very expensive not used by any production code.